### DISCUSSION TITLE
Création de missions primes

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1668] [Nouvelles idées]
C'est un exemple, toutes les données sont modifiables

Après avoir complété 10 missions secondaires à la suite (sans en échouer donc), la prochaine mission secondaire obtenu est une mission dites "prime" ou "supérieure" (j'ai pas réfléchi au nom mdr) qui n'est pas forcément plus dure que les autres (à voir, ça peut être une mission déjà existante ou si vous avez des idées ça peut être l'occasion de créer des missions un peu corsées) et qui est bien plus généreuse qu'une mission secondaire ordinaire. Dans le style 1k_1k500 d'or et 2_3k d'expérience. Ou alors carrément un item au pif (au minimum rare).

**Comment by BastLast** [comment]
j'aime beaucoup l'idée, ça inviterait les gens à faire toutes les missions secondaires et pas seulement les plus rentables :)

**Reply by KirIcare** [reply]
En plus on a ajouté des pts en récompense, si jamais on met en place la suggestion et que le nombre de pts pour la mission prime est plus conséquent qu'une mission classique ça va attirer

**Reply by BastLast** [reply]
Cette idée sera probablement implémentée après le pve
